test/README.md

```diff
@@ -69,3 +69,32 @@ Examples:
 EXPECT_THAT(response->headers(), IsSubsetOfHeaders(allowed_headers));
 EXPECT_THAT(response->headers(), IsSupersetOfHeaders(required_headers));
 ```
+
+## Controlling time in tests
+
+In Envoy production code, time and timers are managed via
+[`Event::TimeSystem`](https://github.com/envoyproxy/envoy/blob/master/include/envoy/event/timer.h),
+which provides a mechanism for querying the time setting up time-based callbacks. Bypassing
```

Comment:
I think this should be "querying the time _and_ setting up"

Reply:
Thanks Greg; done.